versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/configuration-parameters.mdx

```diff
@@ -203,7 +203,7 @@ configured in the [slice configuration YAML file](/versioned_docs/version-0.2.0/
 |Parameter|Parameter Type|Description|Required|
 |----|----|----|----|
 | queueType | String | It is the slice traffic control queue type. The value must be Hierarchical Token Bucket (`HTB`). HTB facilitates guaranteed bandwidth for the slice traffic. | Mandatory |
-| priority | Integer | QoS profiles allows traffic management within a slice as well as prioritization across slices. The value range is 0-3. Integer 0 represents the lowest priority and integer 3 represents the highest priority. | Mandatory |
+| priority | Integer | QoS profiles allows traffic management within a slice as well as prioritization across slices. The value range is 0-3. 0 is the highest priority, and 3 is the lowest priority. | Mandatory |
```

Comment:
change `0 is / 3 is` to `0 represents/ 3 represents`  

A comma after and was not there before. Why did you add it now?

Reply:
@uma-kt  - next time - you respond to the comment.